routes/index.js

```diff
@@ -27,5 +34,17 @@ module.exports = function () {
     })
   })
 
+  router.get('/search', function (req, res) {
+    res.render('search.html', {
+
```

Comment:
why not titles for this one as well? like its done for home and dashboard 
Same with `/owner/name` and `/owner`